op-service/txmgr/txmgr_test.go

```diff
@@ -336,13 +366,11 @@ func TestTxMgrConfirmsMinGasPriceAfterBumping(t *testing.T) {
 
```

Comment:
Does this existing test `TestTxMgrConfirmsMinGasPriceAfterBumping` test the added functionality of the `SimpleTxManager`? If not, we should have an additional test that explicitly covers the branch where at least one gas price increase happens.

Reply:
Yes, the unit tests fail if you do not increase the gas price in the resubmit loop.